attendance/resources/producer.py

```diff
@@ -22,7 +21,10 @@ def connect_kafka_producer():
 
 
 def publish_message(data):
-    if current_app.config['SAGA_ENABLED'] == 1:
+    saga_enabled = os.getenv('SAGA_ENABLED')
+    if saga_enabled is None:
+        saga_enabled = current_app.config['SAGA_ENABLED']
+    if saga_enabled == 1:
```

Comment:
We do not convert os.getenv('SAGA_ENABLED') to integer before doing this comparison.

Could we test setting SAGE_ENABLED environment variable to say 0, config entry of SAGA_ENABLED to 1, and see if 0 takes precedence?

Reply:
tested and changed accordingly